internal/mode/static/nginx/runtime/manager.go

```diff
@@ -17,18 +17,36 @@ import (
 )
 
 const (
-	pidFile            = "/var/run/nginx/nginx.pid"
+	PidFile            = "/var/run/nginx/nginx.pid"
```

Comment:
Let's add comments to all the newly exported variables and types.

Reply:
done